cmd/validate.go

```diff
@@ -79,6 +79,7 @@ var validateCmd = &cobra.Command{
 			}
 			//panic("please see errors above")
 		}
+		return err
```

Comment:
Which `err` is this supposed to be?

Reply:
Yeah now that I think about it, returning `nil` here would make more sense considering we are just printing out warnings in the current implementation and not actually `panic`ing.